Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```
@@ -646,7 +646,7 @@
   components:
   - type: Sprite
     layers:
-    - state: yellow
+    - state: purple
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)